### ISSUE TITLE
When switching to a new page, voiceover doesn't announce the new page title or header

### ISSUE THREAD

**Issue by smcmurtry** [cds-snc/vac-benefits-directory#7]
<a href="https://github.com/smcmurtry"><img src="https://avatars0.githubusercontent.com/u/5498428?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [smcmurtry](https://github.com/smcmurtry)**
_Tuesday Nov 27, 2018 at 16:50 GMT_
_Originally opened as https://github.com/veteransaffairscanada/vac-benefits-directory/issues/1478_

----

This happens both with next/link and with `setSection` `onClick` events in the guided experience.

**Comment by smcmurtry** [comment]
<a href="https://github.com/smcmurtry"><img src="https://avatars0.githubusercontent.com/u/5498428?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [smcmurtry](https://github.com/smcmurtry)**
_Tuesday Nov 27, 2018 at 20:54 GMT_

----

![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Screenreader Navigation](https://trello.com/c/knKtuM4T/100-accessibility-screenreader-navigation)

**Comment by smcmurtry** [comment]
<a href="https://github.com/brdunfield"><img src="https://avatars2.githubusercontent.com/u/5032149?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [brdunfield](https://github.com/brdunfield)**
_Tuesday Jan 08, 2019 at 20:45 GMT_

----

@smcmurtry was this closed by #1649?

**Comment by smcmurtry** [comment]
<a href="https://github.com/smcmurtry"><img src="https://avatars0.githubusercontent.com/u/5498428?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [smcmurtry](https://github.com/smcmurtry)**
_Wednesday Jan 16, 2019 at 21:32 GMT_

----

@brdunfield no that PR just solved the issue for /benefits-directory. The issue still exists for all the other pages.

**Comment by smcmurtry** [comment]
<a href="https://github.com/smcmurtry"><img src="https://avatars0.githubusercontent.com/u/5498428?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [smcmurtry](https://github.com/smcmurtry)**
_Thursday Jan 31, 2019 at 16:18 GMT_

----

Now that the GE has been split into pages this should be easier to solve. I've noticed that clicking the back link (not browser back) has the correct behaviour where the page title is announced and focus goes to the top of the page.

**Comment by smcmurtry** [comment]
<a href="https://github.com/smcmurtry"><img src="https://avatars0.githubusercontent.com/u/5498428?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [smcmurtry](https://github.com/smcmurtry)**
_Thursday Jan 31, 2019 at 21:03 GMT_

----

I've now tried putting autofocusing headers into the GE, but this doesn't seem to affect the voiceover focus ([which is different from the keyboard focus](https://webaim.org/discussion/mail_thread?thread=8526) btw). This is weird because I based the autofocusing off of what IRCC did and theirs does seem to affect the voiceover focus: https://vancouver.rescheduler-dev.cds-snc.ca/register

**Comment by smcmurtry** [comment]
<a href="https://github.com/JuliannaR"><img src="https://avatars2.githubusercontent.com/u/35537308?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [JuliannaR](https://github.com/JuliannaR)**
_Wednesday Mar 06, 2019 at 21:04 GMT_

----

@smcmurtry if your having troubles, something could be a bug with voiceover, can you try validating with NVDA? If not, send it my way and I can see if I can help.

**Comment by smcmurtry** [comment]
<a href="https://github.com/smcmurtry"><img src="https://avatars0.githubusercontent.com/u/5498428?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [smcmurtry](https://github.com/smcmurtry)**
_Thursday Mar 07, 2019 at 16:55 GMT_

----

@JuliannaR If you could try to reproduce this issue with another screen reader that would be a huge help! I only have access to voiceover currently.
